CodeEditModules/Modules/AppPreferences/src/Views/AppearenceChangeView.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  AppearenceChangeView.swift
+//  
+//
+//  Created by 朱浩宇 on 2022/3/30.
+//
+
+import SwiftUI
+import Preferences
+
+struct AppearenceChangeView: View {
+    @StateObject var model = KeyModel.shared
+
+    @AppStorage(Appearances.storageKey)
+    private var appearance: Appearances = .default
+
+    var body: some View {
+        HStack {
```

Comment:
Very repetitive code. Use a subview that can adapt.

Reply:
Fixed